File: src/com/company/superclass/Discipline.java
package com.company.superclass;

public interface Discipline {
    String getName();
    void setName(String name);
    Long getLectureHours();
    void setLectureHours(Long lectureHours);
    int getPracticeHours();
    void setPracticeHours(int practiceHours);
    long getSelfStudyHours();
    void setSelfStudyHours(long selfStudyHours);
    Integer getLabsCount();
    void setLabsCount(Integer labsCount);
    String getDiscipline(Boolean csv);
}

File: src/com/company/superclasses/Discipline.kt
package com.company.superclasses

import kotlin.properties.Delegates

interface Discipline {
        fun getDiscipline(cvs: Boolean = false): String
        var name: String
        var lectureHours: Long?
        var practiceHours: Int
        var selfStudyHours: Long
        var labsCount: Int
}